src/main/java/org/ossgang/commons/observables/weak/WeakMethodReferenceObserver.java

```diff
@@ -57,14 +57,18 @@
  */
 class WeakMethodReferenceObserver<C, T> implements Observer<T> {
 
+    private static final int DEFAULT_CLEANUP_PERIOD = 30;
     private static final ScheduledExecutorService REFERENCE_CLEANUP_EXECUTOR = newSingleThreadScheduledExecutor(
             daemonThreadFactoryWithPrefix("ossgang-weak-observer-cleanup"));
     private static final Set<WeakCleaner> REFERENCE_CLEANERS = Collections.newSetFromMap(new ConcurrentHashMap<>());
 
     static {
+        Integer cleanupPeriod = Maybe.attempt(() -> Integer.parseInt(System.getProperty("ossgang-weak-reference-cleanup-seconds")))
```

Comment:
Perhaps use `Integer.getInteger` with a default value?
Also, should we call the system property something like `org.ossgang.observable.weak_cleanup_period`, i.e. following our package naming (I think this is a bit more common...)?

Reply:
Perfect for the Integer.getInteger! 
Ok for the naming convention, no strong preference there :wink: